src/main/java/baseball/view/OutputView.java

```diff
@@ -0,0 +1,57 @@
+package baseball.view;
+
+import baseball.domain.GameResult;
+import baseball.domain.TryResult;
+
+public class OutputView {
+
+    private static final String START_MESSAGE = "숫자 야구 게임을 시작합니다.";
+    private static final String GAMEOVER_MESSAGE = "3개의 숫자를 모두 맞히셨습니다! 게임 종료";
+    private static final String BLANK = "";
+    private static OutputView instance;
+
+    public static OutputView getInstance() {
+        if (instance == null) {
+            instance = new OutputView();
+        }
+        return instance;
+    }
+
+    public void printStart() {
+        System.out.println(START_MESSAGE);
+    }
+
+    public void printResult(GameResult gameResult) {
+        StringBuilder result = new StringBuilder();
+        result.append(getMessageOf(TryResult.BALL, gameResult));
+        result.append(insertSpace(gameResult));
+        result.append(getMessageOf(TryResult.STRIKE, gameResult));
+        result.append(getNothingMessage(gameResult));
+        System.out.println(result);
```

Comment:
StringBuilder 를 사용하신 이유가 있을까요? 
성능 문제라면, 정말 성능 상의 이점이 있는지 확인해 볼 필요가 있을 것 같습니다.

만약 가독성을 위해서 썼다면 메서드 체이닝도 방법이 될 수 있을 것 같습니닷! ㅇㅅㅇ

```suggestion
        StringBuilder result = new StringBuilder()
                .append(getMessageOf(TryResult.BALL, gameResult))
                .append(insertSpace(gameResult))
                .append(getMessageOf(TryResult.STRIKE, gameResult))
                .append(getNothingMessage(gameResult));

                 ...
```

Reply:
매우 반복되는 작업의 경우에는 StringBuilder가 성능 면에서는 더 좋다고 생각합니다! 
물론 지금 정도의 문자열 처리에서는 크게 차이가 없겠지만요!ㅎㅎ

메서드 체이닝으로 코드 짤 때 아래처럼 해봤다가 '왜 가독성이 왜 더 안좋지?' 하고 안했는데 
```java
StringBuilder result = new StringBuilder().append(getMessageOf(TryResult.BALL, gameResult)).append(insertSpace(gameResult))
```
생각해보니 해로님처럼 줄바꿈을 하면 되는 간단한 문제였군요😂

Reply:
이 부분 같은 경우 반복되는 작업에서 StringBuilder 가 성능이 더 좋은지 확실히 하고 갈 필요가 있습니다! StringBuilder, StringBuffer, String 에 대해 찾아보셔도 좋을 것 같아요!